libultraship/libultraship/SDLController.cpp

```diff
@@ -163,8 +163,15 @@ namespace Ship {
             ay *= scale;
         }
 
-        wStickX = +ax;
-        wStickY = -ay;
+        if (!rightStick) {

```

Comment:
The bool was renamed, so this has to be updated
```suggestion
        if (!isRightStick) {

```